web/src/message_list_view.js

```diff
@@ -1086,6 +1086,13 @@ export class MessageListView {
             }
         }
 
+        const is_conversation_view = message_lists.current.is_combined_feed_view;
+        if (is_conversation_view) {
+            $(".focused-message-list").addClass("combined-feed-view");
```

Comment:
The class name seems a bit misleading since you're only referring to a message in the combined feed view, as opposed to the whole view.

Reply:
I wasn't quite sure with the `combined-feed-view` naming, but this isn't referring to a single message in the view, but the whole messagelist in the combined feed view